yearn/test/system/saddle-strategy.test.js

```diff
@@ -0,0 +1,195 @@
+const { expect } = require("chai")
+const {
+  resetFork,
+  to1ePrecision,
+  impersonateAccount,
+  increaseTime,
+} = require("../helpers/contract-test-helpers.js")
+const { yearn, tbtc, forkBlockNumber } = require("./constants.js")
+const { deployYearnVault } = require("./functions.js")
+const { saddleStrategyFixture } = require("./fixtures.js")
+
+const describeFn =
+  process.env.NODE_ENV === "system-test" ? describe : describe.skip
+
+describeFn("System -- saddle strategy", () => {
+  let vaultGovernance
+  let vaultDepositor
+  let saddlePoolGovernance
+  let saddlePool
+  let tbtcSaddlePoolLPToken
+  let vault
+  let strategy
+
+  before(async () => {
+    await resetFork(forkBlockNumber)
+
+    // Setup roles.
+    vaultGovernance = await ethers.getSigner(0)
+
+    vaultDepositor = await impersonateAccount(
+      tbtc.saddlePoolLPTokenHolderAddress,
+      vaultGovernance
+    )
+
+    saddlePoolGovernance = await impersonateAccount(
```

Comment:
When we change `tbtc.saddlePoolOwner` this should be `saddleLPRewardsGovernance`. Don't confuse Saddle pool with Saddle LP rewards contract.

Reply:
Done in https://github.com/keep-network/tbtc-v2/commit/045e79a6b12147241e18295132a5992350505be2.